include/bolgenos-ng/stdtypes.hpp

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
:grey_exclamation: Combine functionality of this header instead with `int_types.hpp`